File: src/main/java/com/hedvig/backoffice/web/dto/TraceInfoDTO.java
package com.hedvig.backoffice.web.dto;

import lombok.Data;

import java.time.LocalDateTime;
import java.util.UUID;

@Data
public class TraceInfoDTO {
  private LocalDateTime date;
  private String oldValue;
  private String newValue;
  private String fieldName;
  private UUID productId;
  private Long memberId;
  private String userId;
  private Boolean isSuccess;
}

File: src/main/java/com/hedvig/backoffice/web/dto/TraceInfoDTO.kt
package com.hedvig.backoffice.web.dto

import java.time.LocalDateTime
import java.util.UUID

data class TraceInfoDTO(
    val date: LocalDateTime? = null,
    val oldValue: String? = null,
    val newValue: String? = null,
    val fieldName: String? = null,
    val productId: UUID? = null,
    val memberId: Long? = null,
    val userId: String? = null,
    val isSuccess: Boolean? = null
)
